### PR TITLE
Ixspd1 1678  issue fixed Cannot approve We cannot approve the Base token until switching between the options

### PR DESCRIPTION
**Pull request by kapildave07**
## Description

Ixspd1 1678  issue fixed Cannot approve We cannot approve the Base token until switching between the options

## Changes

- Handle all asset token address values
- Create a dynamic dropdown with chainId for different environments

## Attached Links

1. Jira ticket: https://investax.atlassian.net/browse/IXSPD1-1678
2. Documents:

## Checklist

- [x] Manual tests
- [x] The pull request doesn't affect existing feature

## For reviewers

- Ensure new code doesn't break existing features
- Reviewer shouldn't merge if there are pending unresolved comments

### CODE REVIEW

File: src/state/lbp/constants.ts [hunk 1]
```diff
@@ -11,4 +11,10 @@ export const LBP_STAGE_LABELS = [
...
   { label: 'Ended', value: LbpStatus.ended },
-]
\ No newline at end of file
+]
+
+export const NETWORKS_NAME = {
+  USDC: 'USDC',
+  USDT: 'USDT',
+  IXS: 'IXS',
+}
```
Comment (thi-investax) [inline src/state/lbp/constants.ts:20]: why this is called NETWORKS_NAME?

File: src/pages/KYC/mock.ts [hunk 2]
```diff
@@ -32,11 +34,24 @@ export const socialMediaPlatform = ['Telegram', 'Discord', 'X.com', 'Facebook',
...
+  [NetworkName.BASE]: BaseIcon,
+  [NetworkName.POLYGON]: PolygonIcon,
+}
+
+export const blockchainNetworks = Object.entries(NetworkName).map(([key, value]) => {
+  const chainIds = Chains[value]
+  const chainId = ENV_SUPPORTED_TGE_CHAINS?.includes(chainIds[0]) ? chainIds[0] : chainIds[1]
+  return {
+    value,
+    label: value.charAt(0).toUpperCase() + value.slice(1),
```
Comment (thi-investax) [inline src/pages/KYC/mock.ts:50]: @kapildave07 , should write a utility to capitalize first letter. Put it  in `utils/strings`

**Comment by thi-investax** [comment]
please check the comments @kapildave07

File: src/pages/KYC/mock.ts (hunk 2 above)
Comment (kapildave07) [reply src/pages/KYC/mock.ts]: done

File: src/state/lbp/constants.ts (hunk 1 above)
Comment (kapildave07) [reply src/state/lbp/constants.ts]: Instead of NETWORKS_NAME i changed to LBP_NETWORKS

File: src/pages/LBP/components/Tokenomics.tsx [hunk 3]
```diff
@@ -412,6 +412,8 @@ const Tokenomics = ({
       setDirty(true)
     }
   }, [JSON.stringify(formik.touched)])
+
+  console.log(formDataTokenomics.assetTokenAddress, chainId, 'token page')
```
Comment (thi-investax) [inline src/pages/LBP/components/Tokenomics.tsx:416]: remove logs in the pull request

File: src/pages/LBP/components/Approvals.tsx [hunk 4]
```diff
@@ -149,6 +149,8 @@ export default function Approvals({
     )
   }
 
+  console.log(addressA, addressB, chainId, 'approval page')
```
Comment (thi-investax) [inline src/pages/LBP/components/Approvals.tsx:152]: remove logs here

Comment (kapildave07) [reply src/pages/LBP/components/Approvals.tsx]: Resolved

**Review by thi-investax — APPROVED** [review]

**Pull request merged**